src/main/tools/linux-sandbox-options.cc

```diff
@@ -73,6 +73,8 @@ static void Usage(char *program_name, const char *fmt, ...) {
           "  -R  if set, make the uid/gid be root\n"
           "  -U  if set, make the uid/gid be nobody\n"
           "  -D  if set, debug info will be printed\n"
+          "  -h  if set, use whitelisting for improved hermeticity\n"
+          "  -s  The sandbox root directory\n"
```

Comment:
Could you clarify how this is different from the working dir (`-W`)?

Reply:
I guess working dir is enough, if we can assume that working dir will be sandbox_dir + /execroot/workspace_name>, then it can be derived from that.